serokell-util.cabal

```diff
@@ -63,10 +60,11 @@ library
                      , fmt
                      , formatting >= 6.2.0 && < 6.3
                      , hashable >= 1.2.4.0
-                     , lens
                      , log-warper ^>= 1.8.9
+                     , microlens
+                     , microlens-mtl
+                     , microlens-platform
```

Comment:
`microlens-platform` already includes other `microlens` packages so You don't need to add them to `.cabal` if you use `microlens-platform`. Other question: why you need `microlens-platform` at all? :thinking: